addons/website_slides/static/src/slide_category_one2many_field.js

```diff
@@ -14,4 +14,5 @@ SlideCategoryOneToManyField.defaultProps = {
     editable: "bottom",
 };
 
+SlideCategoryOneToManyField.additionalClasses = ["o_field_one2many"];
```

Comment:
Looks like other occurrences of this have fixed it with:

```js
import { X2ManyField, x2ManyField } from "@web/views/fields/x2many/x2many_field";
...
SlideCategoryOneToManyField.additionalClasses = [...x2ManyField.additionalClasses || [], "o_field_one2many"];
```

Which seems a bit cleaner as it will re-use additionalClasses defined by the extended x2many field is defined.
(I don't think there is any currently, but it's cleaner anyway).

Can you give it a try?